style.css

```diff
@@ -191,6 +191,17 @@ ul li {
   margin-bottom: 29px;
 }
 
+.button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+.button:enabled {
+  background: #36b37f;
+  border-radius: 4px;
+}
+
```

Comment:
- [ ] Good job implementing the styling, there is one thing left, please implement the design for the Pressed state which is called in CSS `active`. 😁

Reply:
I have implemented the CSS active.